crates/sui-bridge/src/types.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::collections::HashMap;
+
+use fastcrypto::secp256k1::{Secp256k1PublicKey, Secp256k1Signature};
+use sui_types::multiaddr::Multiaddr;
+
+pub type BridgeCommitteePublicKey = Secp256k1PublicKey;
```

Comment:
IIUC in Sui and Narwhal, most keys are stored as bytes as struct members and only deserialized when it is used.

Reply:
I have the same impression but couldn't find where the code is now. do you know? 
This is how the key is loaded: https://github.com/MystenLabs/sui/blob/main/crates/sui-node/src/lib.rs#L419

Reply:
I was thinking about types like `AuthorityPublicKeyBytes`. Also if this type will not be part of generate_format.rs, it should be ok. Otherwise it will be hard to fix the format test.